### DISCUSSION TITLE
Different analog meter style supported?

### DISCUSSION THREAD

**Discussion by dtrumpfheller** [jomjol/AI-on-the-edge-device#1776] [Q&A] [answered]
Hello,

I started to play with the water meter tool and am pleased so far. But my water meter is quite different to the one from the author with everything positioned within the analog part. Thus I did not expect the analog readout to work and surely it doesn't.
Now I am wondering if the AI would be able to read the analog at all due to the integration of everything else within it? I don't want to spend a lot of time getting images and training the AI if the spinning digit meters within the analog meter would create too much noise(?) to allow a recognition after all. Any suggestions or experience with this?

The analog meter in best case would provide two digits instead of one if possible.
I know the LED right now burns out parts of the the image, still fiddling with this.

![watermeter](https://user-images.githubusercontent.com/398637/210926103-55b307d3-d338-4fb4-8fda-c17d9e9a7dd0.PNG)

**Comment by caco3** [comment, marked as answer]
you could try to make the analog part smaller, maybe 1/3 of its current size. you don't need its number scale, just the red pointer.

but as you pointed out, the reflection must go away also

**Reply by dtrumpfheller** [reply]
good idea, will give it a try

**Comment by ghost** [comment]
> Hello,
> 
> I started to play with the water meter tool and am pleased so far. But my water meter is quite different to the one from the author with everything positioned within the analog part. Thus I did not expect the analog readout to work and surely it doesn't. Now I am wondering if the AI would be able to read the analog at all due to the integration of everything else within it? I don't want to spend a lot of time getting images and training the AI if the spinning digit meters within the analog meter would create too much noise(?) to allow a recognition after all. Any suggestions or experience with this?
> 
> The analog meter in best case would provide two digits instead of one if possible. I know the LED right now burns out parts of the the image, still fiddling with this.
> 
> ![watermeter](https://user-images.githubusercontent.com/398637/210926103-55b307d3-d338-4fb4-8fda-c17d9e9a7dd0.PNG)

The size of the image could be a memory issue. But like @caco3 suggest, you can use a bit smaller size. It could be, the recognition is not good, but you can collect than your own data and learn the model for yourself.  See https://jomjol.github.io/AI-on-the-edge-device-docs/Learn-models-with-your-own-images/
